src/test/java/org/gridsuite/voltageinit/utils/ReportUtilTest.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.voltageinit.utils;
+
+import com.powsybl.commons.report.ReportNode;
+import org.gridsuite.voltageinit.server.VoltageInitApplication;
+import org.gridsuite.voltageinit.server.util.ReportUtil;
+import org.junit.jupiter.api.Test;
+import org.springframework.test.context.ContextConfiguration;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@ContextConfiguration(classes = {VoltageInitApplication.class})
```

Comment:
You can also remove this one. And you should remove public keywords with junit5 for class and test method

Reply:
:+1: 
Done